test/unit/specs/components/Lend/LoanSearch/ChallengeCallout.spec.js

```diff
@@ -4,14 +4,22 @@ import ChallengeCallout from '@/components/Lend/LoanSearch/ChallengeCallout';
 
 const shareLender = { id: 1, name: 'Lender' };
 const teamName = 'Team Test';
+const mocks = {
+	$route: {
+		query: {
+			team: teamName
+		}
+	}
+};
 
 describe('ChallengeCallout', () => {
-	it('should display share lender in message', () => {
+	it('should display team name', () => {
 		const { getByText } = render(ChallengeCallout, {
-			props: { shareLender, teamName }
+			props: { shareLender, teamName },
+			mocks,
 		});
 
-		getByText(`Support ${shareLender.name} and help ${teamName} hit their goal`);
```

Comment:
Don't we still want to check the entire message? The lender name should still be displayed.